sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml

```diff
@@ -113,7 +113,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.3.4</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <version>1.3.5</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
Do we have a bug in the version increment PRs?

Reply:
We might have a bug. Checked the commit history: https://github.com/Azure/azure-sdk-for-java/commits/main/sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml

Seems there is no IncreaseVersion PR commit in this pom file

Reply:
This is again an issue with an old PR with successful CI pipelines merged after a long time. See [discussion here](https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1628550967342?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1628550967342&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1628550967342)